### ISSUE TITLE
[BUG] Accessibility -  Web App & App - Dynamic message not read by screen reader

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5140]
**Describe the bug**
When a user has no Mckinley appointment data, an error is to be expected. 
Using TalkBack on Android this error message appears but is not read to screen reader user. 

**To Reproduce**
1. Enable TalkBack
2. Click on 'Maps'
3. Select 'MyMcKinley...'

**Expected behavior**
When a user has no Mckinley appointment data, an error is to be expected. 
Using TalkBack on Android this error message should be read to the screen reader user. 

**Actual behavior**
The user just hears "Close" button

**Screenshots**

![Image](https://github.com/user-attachments/assets/4652275b-4c12-48bd-ae3a-3fdbe2aa2b38)

**Smartphone (please complete the following information):**
Android:
•	OS: Android Version 15
•	Device: Google Pixel 8a
•	Browser: Chrome 136.0.7103.87

**Additional context**

**Comment by acovher2** [comment]
Retest passed. Messaged received focus and both message and close button announced to TalkBack audience. 
Android:
•	OS: Android Version 15
•	Device: Google Pixel 8a
•	Browser: Chrome 141.0.7390.123
